frontend/services/src/users/usersService.ts

```diff
@@ -46,6 +46,32 @@ export async function getPublicUserById(userId: UserId): Promise<UserData> {
   }
 }
 
+export async function getPrivateUserById(userId: UserId): Promise<UserData> {
+  userServiceLogger.info(`Fetching private user by ID:, ${userId}`);
+  console.log(userId);
+  if (userId === undefined) {
```

Comment:
should we perform a null check here as well?

Reply:
mmmm yeah added || userId === null